spi/common/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/event/policydefinition/PolicyDefinitionEventPayload.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *       Fraunhofer Institute for Software and Systems Engineering - expending Event classes
+ *
+ */
+
+package org.eclipse.dataspaceconnector.spi.event.policydefinition;
+
+import org.eclipse.dataspaceconnector.spi.event.EventPayload;
+
+public class PolicyDefinitionEventPayload extends EventPayload {
```

Comment:
Please include documentation

Reply:
How about: "Contains data related to policy definition events"?

Reply:
I add this additionally to the current information, ok?